src/server/views/lists/partials/translators-interpreters/translators-interpreters-results-list.njk

```diff
@@ -12,22 +12,35 @@
   <p class="govuk-body">We couldn’t find any results that match your search terms. Try changing these to search again.</p>
 {% else %}
   <h2 class="govuk-heading-l govuk-!-margin-bottom-4">{{resultsTitle}}</h2>
-  <p class="govuk-body">You should carry out your own research before choosing a service provider.</p>
 
+  {% if hasSworn.translatorsAndInterpreters or hasSworn.interpreters or hasSworn.translators %}
   <div class="govuk-warning-text">
     <span class="govuk-warning-text__icon" aria-hidden="true">!</span>
     <strong class="govuk-warning-text__text">
       <span class="govuk-warning-text__assistive">Warning</span>
-      {% if hasSworn %}
-        Translators and interpreters must be authorised to operate in some countries (known as sworn translators).
-        Check providers are qualified to carry out the service you need in the country you have selected.
-      {% else %}
-        In {{ country }}, translators must be authorised to carry out official translations of legal documents.
-        <p>These translators are known as 'sworn' or 'certified' translators.</p>
-        <p>Check that a translator is authorised to provide an official translation before you use them.</p>
+      {% if hasSworn.translatorsAndInterpreters or (hasSworn.interpreters and hasSworn.translators) %}
```

Comment:
does this make sense?  Why wouldn't you just have `hasSworn.translatorsAndInterpreters`?

Reply:
Yep, so they way it worked before was that `translatorsAndInterpreters` only applied to list items that have both interpreters and translators. But Neil noticed the message wasn't showing when 1 list item was a translator and another was an interpreter so that's why this case was added. 

But I've now changed the way  `translatorsAndInterpreters`  so it does both ways.